### PR TITLE
Update DCP_change_log.md

### PR DESCRIPTION
**Pull request by KoenAnnys**

### CODE REVIEW

File: release-notes/Cloud/DCP_change_log.md [hunk 1]
```diff
@@ -9,6 +9,27 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 08 August 2024 - Feature - A catalog is limited to have 5 tags [ID_40349]
```
Comment (baptistepa) [inline release-notes/Cloud/DCP_change_log.md:12]: catalog item

File: release-notes/Cloud/DCP_change_log.md [hunk 2]
```diff
@@ -9,6 +9,27 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+
+Registering of a catalog with more than 5 tags will fail and is not supported.
```
Comment (baptistepa) [inline release-notes/Cloud/DCP_change_log.md:14]: catalog item

File: release-notes/Cloud/DCP_change_log.md [hunk 3]
```diff
@@ -9,6 +9,27 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+
+Search results on tags are now case insensitive.
+
+### 08 August 2024 - Feature - Catalog - Search supports sorting on name and type of a catalog item [ID_40368]
+
+In the browse page, a user is able to sort on the name and type of the items in ascending or descending order. By default items will be sorted on name in ascending order.
+
+### 08 August 2024 - Fix - Catalog - Recommended versions are shown correctly for non-connector items [ID_40346]
+
+When a user went to the versions tab of a non-connector they would not see any recommended version. From now on a recommended version will always be shown.
```
Comment (baptistepa) [inline release-notes/Cloud/DCP_change_log.md:31]: Just reword to "Recommend versions are now showing for all catalog item types"

**Review by baptistepa — CHANGES_REQUESTED** [review]

File: release-notes/Cloud/DCP_change_log.md (hunk 3 above)
Comment (KoenAnnys) [reply release-notes/Cloud/DCP_change_log.md]: ok, reworded

File: release-notes/Cloud/DCP_change_log.md (hunk 2 above)
Comment (KoenAnnys) [reply release-notes/Cloud/DCP_change_log.md]: ok, done

File: release-notes/Cloud/DCP_change_log.md (hunk 1 above)
Comment (KoenAnnys) [reply release-notes/Cloud/DCP_change_log.md]: ok, done

**Review by baptistepa — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**